stacktrace.go

```diff
@@ -206,8 +215,7 @@ func filterFrames(frames []Frame) []Frame {
 			continue
 		}
 		// sentry internal frames
-		if frame.Module == "github.com/getsentry/sentry-go" ||
-			strings.HasPrefix(frame.Module, "github.com/getsentry/sentry-go.") {
+		if frame.Module == "github.com/getsentry/sentry-go" {
```

Comment:
Will this be `github.com/getsentry/sentry-go/fasthttp` for integrations? I feel like it should be if we want to be close to Python.

Reply:
Yes it will be, and we should filter it out -- what we don't :( Yet.

I realize #125 was incomplete, we need to filter out frames of all packages belonging to the SDK (keeping the behavior of the latest released version).

Thanks for asking! I'll fix it.

Reply:
Fixed in the latest patch.